### DISCUSSION TITLE
why 3 different Windows releases?

### DISCUSSION THREAD

**Discussion by ClaudeRobbinCR** [LostRuins/koboldcpp#1109] [Q&A] [answered]
The first one is without CUDA, I understand that it runs on the CPU. No GPU acceleration. Got it.
The other one comes with Cu 12. Got that too. It is 600 Mb nearly. 
But why the third one which is 433 MB or so? What is up with that? Does it use Cuda 11 or something?

**Comment by LostRuins** [comment]
The differences are mentioned inside the release notes.

To use, download and run the koboldcpp.exe, which is a one-file pyinstaller.
If you don't need CUDA, you can use koboldcpp_nocuda.exe which is much smaller.
If you have an Nvidia GPU, but use an old CPU and koboldcpp.exe does not work, try koboldcpp_oldcpu.exe
If you have a newer Nvidia GPU, you can use the CUDA 12 version koboldcpp_cu12.exe (much larger, slightly faster).

**Reply by ClaudeRobbinCR** [reply]
Normal installer - 433 MB

Cu 12 - 560 MB

No cuda - 60 MB

MY DOUBT is
The normal installer has WHCIH CUDA VERSION?

Or does it also run on Cuda 12? But has support for older CPUs and where can I get this info in the wiki?

**Reply by askmyteapot** [reply, marked as answer]
> The normal installer has WHCIH CUDA VERSION?

Vanila Koboldcpp: CUDA 11.4.4
CU12 Koboldcpp: CUDA 12.1.0

But, its not in the Wiki or the release description. 
That was taken from the github workflows. [.github/workflows](https://github.com/LostRuins/koboldcpp/blob/concedo/.github/workflows/)

**Reply by ClaudeRobbinCR** [reply]
Thank you teapot!
